### PR TITLE
auto-commit linter changes

### PR DESCRIPTION
**Pull request by smiller171**

### CODE REVIEW

**Comment by sblack4** [comment]
I'm. confused, this looks like it has a lot of the changes from #7

File: .github/workflows/pre-commit-check.yaml [hunk 1]
```diff
@@ -4,19 +4,37 @@ on:
...
-        tfenv install
-    - name: pre-commit run all
-      run: |
-        pre-commit run -a
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Install prerequisites
+        run: ./bin/install-macos.sh
+      - name: initiallize Terraform
```
Comment (cdaniluk) [inline .github/workflows/pre-commit-check.yaml]: typo
```suggestion
      - name: initialize Terraform
```

**Comment by smiller171** [comment]
> I'm. confused, this looks like it has a lot of the changes from #7

It does. This one was branched off of that branch

**Review by cdaniluk — APPROVED** [review]
one suggested change

**Pull request merged**